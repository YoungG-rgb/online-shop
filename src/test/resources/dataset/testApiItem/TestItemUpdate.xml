<dataset>
    <item id="1" count="2" description="Test" discount="1" is_moderate_accept="false" is_moderated="false"
          pretendent_to_be_deleted="false" moderated_reject_reason="1" name="Test" price="2000.00" rating="0"/>
    <shop id="1" activity="1" count="1" description="Test"  email="Test@mail" is_moderate_accept="false"
          is_moderated="false" is_pretendent_to_be_deleted="false" moderated_reject_reason="Test"  name="Alibaba"
          phone="Test" rating="0"/>
    <category id="1" name="Sport"/>
    <shop_item shop_id="1" item_id="1" />
</dataset>