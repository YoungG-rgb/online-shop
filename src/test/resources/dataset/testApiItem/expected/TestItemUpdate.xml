<dataset>
    <item id="1" count="2" description="Test" is_moderate_accept="false" is_moderated="false"
          pretendent_to_be_deleted="false" moderated_reject_reason="1" name="TestTest" price="2000.00" rating="0"/>
</dataset>